doc/api/errors.md

```diff
@@ -448,13 +448,16 @@ system error.
 
 ### error.errno
 
-* {string|number}
+* {number}
+
+The `error.errno` property is a negative number which corresponds
+to the error code defined in [`libuv Error handling`].
+See the libuv `errno.h` header file (`deps/uv/include/uv/errno.h`
```

Comment:
Instead of pointing users at the libuv source code, can we point them at http://docs.libuv.org/en/v1.x/errors.html.

Reply:
That's where the [`libuv Error handling`] above points to, though I have no idea why this adds another link to the source, maybe it's just historical.

Reply:
I'd just drop the "See the libuv errno.h..." sentence completely.